posthog-ai/package.json

```diff
@@ -22,11 +22,13 @@
     "ai",
     "openai",
     "anthropic",
+    "gemini",
     "llm",
     "observability"
   ],
   "dependencies": {
     "@anthropic-ai/sdk": "^0.36.3",
+    "@google/genai": "^1.1.0",
     "@langchain/core": "^0.3.37",
     "ai": "^4.1.0",
     "langchain": "^0.3.15",
```

Comment:
Random question: does it make sense to *require* these as dependencies? Shouldnt they be peer dependencies instead?

Reply:
Good point. But will be a breaking change. I'll fix CI, merge this and do a follow up and bump the major Version

Reply:
Yeah, good call